integration-tests/wallet-big-map.spec.ts

```diff
@@ -7,15 +7,15 @@ import { storage } from '../packages/taquito-michelson-encoder/data/proto005/uni
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
 
-  describe(`Test wallet with multiple bigmap variations using: ${rpc}`, () => {
+  describe(`Test contract origination with initialized BigMap with variants of data through wallet api using: ${rpc}`, () => {
```

Comment:
There was a mistake, it is not bigmaps but maps in the contract

Reply:
Changed to map.  Also cleaned up unused imports